css/main.scss

```diff
@@ -12,3 +12,30 @@
         "syntax-highlighting",
         "github"
 ;
+
+h2{
```

Comment:
Can you move this into it's own partial, since it's specific to anchor css rather than the webpage.